### PR TITLE
added graphviz code to generate pipeline infographic

### PR DESCRIPTION
**Pull request by psteinb**
this adds the following humble visualization to the introduction:

pipeline

There are a lot of things to improve. But I think it is a helpful tool for learners.

### CODE REVIEW

**Comment by colinsauze** [comment]
Should we have an arrow from the "create model or use pretrained model"  box going to "Perform prediction" for when a pretrained model is used?

**Comment by psteinb** [comment]
As an alternative, I used the following adaptation from above during a workshop recently:
image

**Comment by dsmits** [comment]
@psteinb why do you have identify inputs and outputs -> prepare data and create model -> choose loss and optimizer in parallel? Because you can do them at the same time? They don't really exclude one another.

**Comment by dsmits** [comment]
@colinsauze it does make sense to skip "train the model" when you are using a pretrained model. Although then you would have to make an exception if you're finetuning.

I think it's best to keep things simple so my vote is for keeping it serialized. It's more like a checklist. People will know they can skip training the model when they already have a trained model.

**Comment by psteinb** [comment]
done with [0abeb76](https://github.com/carpentries-incubator/deep-learning-intro/pull/153/commits/0abeb7626694a887e6e6682edac4475d27dd9958)

**Review by dsmits — APPROVED** [review]
